app/human-detector-app/classes/User.tsx

```diff
@@ -35,4 +37,19 @@ export default class User {
     this.groupList.splice(groupIndex, 1);
     return groupRemoved;
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  getAllNotifications(allNotifications: Notification[], groups: Group[]): Notification[] {
```

Comment:
This method shouldn't need to take any parameters I don't think.  It should just be `getAllNotifications()` which will return a `Notification[]`.  That should get rid of all of the eslint comments that you added in this file.

In other words, instead of passing `groups` in the function, we should be just using `this.groupList` in the User class.